app/src/main/res/layout/main_fragment.xml

```diff
@@ -10,22 +10,73 @@
     <AutoCompleteTextView
         android:id="@+id/actFurnitureName"
         android:layout_width="0dp"
-        android:layout_height="wrap_content"
+        android:layout_height="45dp"
```

Comment:
Keep this as it were.  Better to have a fluid, flexible height (wrap_content) than a fixed one (45dp).  Do this to make your app look good on many resolutions.